src/test/java/posters/pageobjects/components/MiniCart.java

```diff
@@ -46,7 +46,10 @@ public void openMiniCart()
         headerCart.scrollTo().click();
         // Wait for mini cart to appear
         // Wait for the mini cart to show
-        miniCart.waitUntil(visible, Neodymium.configuration().selenideTimeout());
+        // Sometimes there can be a timeout at this point.
+        // Then you can increase the value from default 3s to 5 seconds.
+        miniCart.waitUntil(exist, Neodymium.configuration().selenideTimeout());
+        // miniCart.waitUntil(visible, Neodymium.configuration().selenideTimeout());
```

Comment:
this is the open mini cart  function we need to validate that it is visible. please revert this change since a check for existence is not sufficient.

Reply:
done